backend/src/services/calendly.ts

```diff
@@ -32,4 +32,42 @@ async function getCalendlyEventDate(uri: string, accessToken: string) {
   }
 }
 
-export { getCalendlyEventDate };
+async function validateCalendlyLink(link: string) {
+  if (!link.startsWith("https://calendly.com/")) {
+    throw ValidationError.INVALID_CALENDLY_LINK;
```

Comment:
It might be good to have this be a separate error type so it's easier to debug on frontend.